docs/.redirects/redirects.json

```diff
@@ -19,7 +19,6 @@
     "training/apis-howto/deepspeed/deepspeed": "../../../model-dev-guide/apis-howto/deepspeed/deepspeed.html",
     "training/apis-howto/deepspeed/overview": "../../../model-dev-guide/apis-howto/deepspeed/overview.html",
     "training/apis-howto/deepspeed/pytorch2deepspeed": "../../../model-dev-guide/apis-howto/deepspeed/pytorch2deepspeed.html",
-    "training/apis-howto/api-pytorch-lightning-ug": "../../model-dev-guide/apis-howto/api-pytorch-lightning-ug.html",
```

Comment:
No deleting from redirects file.  Instead change where they redirect to point to somewhere useful.

This should fail CI at some point, but maybe not till the release is going to be cut.

Let's chat and figure this out synchronously.